packages/api/src/utils/stripe.js

```diff
@@ -235,19 +292,55 @@ export function createMockStripeCardPaymentMethod () {
   }
 }
 
-/** @returns {StripeComForCustomersService} */
+/**
+ * Create a mock stripe sdk instance that can be used with the StripeCustomersService
+ * @returns {StripeComForCustomersService}
+ */
 export function createMockStripeForCustomersService () {
```

Comment:
this looks good and makes sense, AND is starting to get to the level of complexity where it seems worth investigating whether we can use Stripe's API in "test mode" (https://stripe.com/docs/keys#test-live-modes) to get some end-to-end testing into place. 

I dug around in the Stripe API docs enough to convince myself it would probably be worth investigating this but not enough to be sure they actually implement this particular API functionality in test mode - I would definitely be down to do a quick spike on this if you think it makes sense to keep digging into this!

Reply:
adding - if I do find this is possible, I'd want to add it to a test suite that primarily gets run during CI so we can still run the existing tests offline/on a plane. it seems like `test:e2e` in the top level `package.json` might be a good place for this - seems like I'd need to add a similar `test:e2e` script to the `api` project, does that sound right?